warzone/GameEngine.cpp

```diff
@@ -69,7 +69,7 @@ void GameEngine::startUpPhase() {
 		cout << "What map file would you like?";
 		cin >> mapFileName;
 		loader = new MapLoader(mapFileName);
-		if (loader->validateMapFormat() == true)
+		if (loader->validateMapFormatML() == true)
```

Comment:
Can we add some logic here to either

- ask the user to pick which maploader to use, or
- try the Dominion one, if it returns invalid try the Conquest one, if neither work then ask the player to enter a new name

I think the better option would be the 2nd one